broadcast/src/linked/signer/actor.rs

```diff
@@ -548,11 +531,10 @@ impl<
                 .await;
         }
 
-        // Ignore sending the verification request to the application
-        // if the number of pending requests is too high.
-        // We use > instead of >= because the stream always has one dummy future.
-        if self.pending_verifies.len() > self.pending_verify_size {
-            warn!(n=?self.pending_verifies.len(), "too many pending verifies");
+        // Drop the node if there are too many pending verifies
+        let n = self.pending_verifies.len();
```

Comment:
Like with `resolver`, I think this should be limited by the channel rate limit + mailbox size?

Reply:
It'll be a great metric to track for sure 👀

Reply:
If so, please create an issue and link it here 🙏

Reply:
https://github.com/commonwarexyz/monorepo/issues/534